docker/README.md

```diff
@@ -72,15 +72,15 @@ sudo docker run -ti -e TERM=linux --name gitlab-ce-troubleshoot --publish 8080:8
 ### Get published images from Dockerhub
 
 ```bash
-sudo docker pull sytse/gitlab-data
+sudo docker pull sytse/gitlab_data
```

Comment:
Great :) I changed the image name back to `sytse/gitlab-data` and squashed the commits! I hope everything is fine now :)